src/Symfony/Component/JsonPath/JsonPathUtils.php

```diff
@@ -85,4 +85,80 @@ public static function findSmallestDeserializableStringAndPath(array $tokens, mi
             'tokens' => $remainingTokens,
         ];
     }
+
+    public static function unescapeString(string $str, string $quoteChar): string
+    {
+        if ('"' === $quoteChar) {
+            // try JSON decoding first for unicode sequences
+            $jsonStr = '"' . $str . '"';
```

Comment:
looks like fabbot has an issue these days, it misses many small things like this one:
```suggestion
            $jsonStr = '"'.$str.'"';
```